### PR TITLE
feature/COR-1502-remove-redirects-vr-pages

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard21**
## Summary

* Removed VR related redirects

### CODE REVIEW

File: packages/app/src/next-config/redirects/redirects.js [hunk 1]
```diff
@@ -42,43 +42,13 @@ async function redirects() {
...
-      source: '/veiligheidsregio/:gm(gm|GM|gM|Gm):nr(\\d{4}):slash(/{0,1}):page*',
-      destination: '/gemeente/GM:nr',
-      permanent: false,
-    },
-    {
-      source: '/veiligheidsregio/:code/risiconiveau',
-      destination: '/veiligheidsregio/:code',
-      permanent: true,
-    },
     // Redirects for former topical pages GM/VR -> dedicated GM/VR page
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/next-config/redirects/redirects.js:45]: Maybe remove VR from the comment as well?

Comment (VWSCoronaDashboard21) [reply packages/app/src/next-config/redirects/redirects.js]: You are right, I have removed VR from the comment. Please resolve this accordingly.

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice PR

File: packages/app/src/next-config/redirects/redirects.js [hunk 2]
```diff
@@ -42,43 +42,13 @@ async function redirects() {
...
-      permanent: false,
-    },
-    {
-      source: '/gemeente/:vr(vr|VR|vR|Vr):nr(\\d{2}):slash(/{0,1}):page*',
-      destination: '/veiligheidsregio/VR:nr',
-      permanent: false,
-    },
-    {
-      source: '/veiligheidsregio/:gm(gm|GM|gM|Gm):nr(\\d{4}):slash(/{0,1}):page*',
-      destination: '/gemeente/GM:nr',
```
Comment (VWSCoronaDashboard25) [inline packages/app/src/next-config/redirects/redirects.js:57]: There is no reason to remove the redirect to the GM page. By removing this, someones bookmark following an old structure to their hometown will not work anymore, although the hometown page is still up.

Comment (VWSCoronaDashboard21) [reply packages/app/src/next-config/redirects/redirects.js:57]: This has been discussed with team Communication and the solution will be picked up in another ticket. See comment in the related ticket of this PR. Please resolve accordingly.

Comment (VWSCoronaDashboard21) [reply packages/app/src/next-config/redirects/redirects.js:57]: Thanks for updating, I have added this redirect back again. Please resolve accordingly.

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice! Good stuff!

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Review by VWSCoronaDashboard25 — APPROVED** [review]

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Pull request merged**